stdlib/public/Platform/tgmath.swift.gyb

```diff
@@ -20,11 +20,20 @@ public func fabs<T: FloatingPoint>(_ x: T) -> T {
 }
 
 @_transparent
+@differentiable(
```

Comment:
```suggestion
// SWIFT_ENABLE_TENSORFLOW
@differentiable(
```

Reply:
done in 27ca6a252ac2f46f4fe6c3f425dab2138fd439f3